Lib/Src/CLog.cpp

```diff
@@ -47,30 +47,30 @@ void CLog::log(log_level_t log_level, const std::string &message)
             "[ERROR]->" + m_module_name + ": " + message + "\n"));
 
         // Send message using UART
-        mp_error_uart->sendMessage(full_msg);
+        mp_error_uart->send(full_msg);
```

Comment:
Again, you are modifying too many things in one PR. It gets confusing what the purpose of this branch is.

Reply:
ah... this is because the uart method changed...